docs/guides/transfer_learning.ipynb

```diff
@@ -92,7 +92,7 @@
     "\n",
     "Calling `load_model` from the snippet above returns the `FlaxCLIPModule`, which is composed of `text_model` and `vision_model` submodules.\n",
     "\n",
-    "An easy way to extract the `vision_model` submodule defined inside `.setup()` and its variables is to use [`nn.apply`](https://flax.readthedocs.io/en/latest/api_reference/flax.linen.html#flax.linen.apply) to run a `extract_submodule` helper function, inside this function the `clip` Module is bounded to its variables and fields defined in `.setup()` are accessible:"
+    "An easy way to extract the `vision_model` submodule defined inside `.setup()` and its variables is to use [`bind`](https://flax.readthedocs.io/en/latest/api_reference/flax.linen.html#flax.linen.Module.bind) on the `clip` Module immediatly followed by [`unbind`](https://flax.readthedocs.io/en/latest/api_reference/flax.linen.html#flax.linen.Module.unbind) on the `vision_model` submodule."
```

Comment:
- Using a full method name (`flax.linen.Module.bind`) can help some (new) users identify what is what.
- A minor typo fix.
- "sub-Module".

```suggestion
    "An easy way to extract the `vision_model` sub-Module defined inside `.setup()` and its variables is to use [`flax.linen.Module.bind`](https://flax.readthedocs.io/en/latest/api_reference/flax.linen.html#flax.linen.Module.bind) on the `clip` Module immediately followed by [`flax.linen.Module.unbind`](https://flax.readthedocs.io/en/latest/api_reference/flax.linen.html#flax.linen.Module.unbind) on the `vision_model` sub-Module."
```